### ISSUE TITLE
LibreCAD is missing default/built-in plugins

### ISSUE THREAD

**Issue by xyzz** [void-linux/void-packages#32497]
### System

* xuname:  
  Void 5.12.19_1 x86_64 AuthenticAMD uptodate hold rrrrmdFFFFFFFF
* package:  
  *affected package(s) including the version*: LibreCAD-2.1.3_3

### Expected behavior

LibreCAD has several default plugins documented in https://librecad.readthedocs.io/en/latest/ref/menu.html#plugins - these should be accessible via the "Plugins" menu. To confirm this, I've installed LibreCAD 2.1.3 on windows using their official installer and the following plugins are present:

![win-plugins](https://user-images.githubusercontent.com/1065521/129461042-39a78fce-1f15-4b18-8f1f-5b14c0a9831e.png)

### Actual behavior

After installing LibreCAD from Void Linux repos, the "Plugins" menu is empty (this is after trying to click on the menu):

![image](https://user-images.githubusercontent.com/1065521/129460975-c77e97e3-7658-47b7-bce1-d5ece6097595.png)

### Steps to reproduce the behavior

1. Install and launch LibreCAD
2. Click on "Plugins" in the menu bar

**Comment by sburris0** [comment]
Looks like the Debian package copies two things:
1. `unix/librecad` to `/usr/bin`
2. `unix/resources/plugins/*` to `/usr/lib/librecad`

The Void template only does the first.

I think the template should contain something like:

```bash
# install plugins
for plugin in plugins; do
    vcopy unix/resources/${plugin} usr/lib/librecad/
done
```

though I can't test because `harfbuzz` build is failing for me.

**Comment by ericonr** [comment]
It was an accidental push, i believe.

**Comment by abenson** [comment]
It was.

Debian also symlinks the plugins to `usr/share`.  I still need to figure out how to change the load path.  I'll be back at it tonight.

**Comment by hervyqa** [comment]
Solved #34683
